va_explorer/va_data_management/management/commands/run_coding_algorithms.py

```diff
@@ -1,15 +1,52 @@
+import argparse
+import pandas as pd
+import time
 from django.core.management.base import BaseCommand
-from va_explorer.va_data_management.utils.coding import run_coding_algorithms
+from django.db.models import F
+from va_explorer.va_data_management.utils.coding import run_coding_algorithms, ALGORITHM_SETTINGS, validate_algorithm_settings
+from va_explorer.va_data_management.models import VerbalAutopsy, CauseOfDeath
 
 
 class Command(BaseCommand):
 
     help = 'Run cause coding algorithms'
 
-    def handle(self, *args, **options):
-        results = run_coding_algorithms()
-        num_coded = len(results['causes'])
-        num_total = len(results['verbal_autopsies'])
-        num_issues = len(results['issues'])
+    def add_arguments(self, parser):
+        # whether or not to overwrite existing CODs - if True, will clear all CODs before running
+        parser.add_argument('--overwrite', type=bool, nargs='?', default=False)
+        parser.add_argument('--cod_fname', type=str, nargs='?', default='old_cod_mapping.csv')
 
-        self.stdout.write(f'Coded {num_coded} verbal autopsies (out of {num_total}) [{num_issues} issues]')
+    def handle(self, **options):
+        ti = time.time()
+        # validate algorithm settings first. Only proceed if settings are valid. 
+        settings_valid = validate_algorithm_settings(ALGORITHM_SETTINGS)
+        if settings_valid:
```

Comment:
Probably don't need `settings_valid` variable since it's only being using in line 23. 

Can just do:
`if validate_algorithm_settings(ALGORITHM_SETTINGS)`: